packages/flutter_tools/lib/src/vmservice.dart

```diff
@@ -1314,6 +1314,16 @@ class FlutterView extends ServiceObject {
     await subscription.cancel();
   }
 
+  Future<Null> setAssetDirectory(Uri assetsDirectory) async {
+    if (assetsDirectory != null) {
```

Comment:
when will the assetsDirectory be null?
I am wondering if it is sufficient to assert that assetsDirectory is not null.

Reply:
Good point, done.